app/og/route.tsx

```
@@ -8,153 +11,199 @@
+    if (dateString.includes(" ")) {
+      // Handle the specific format from the URL
+      const [datePart, timePart] = dateString.split(" ");
+      const [year, month, day] = datePart.split("-");
+      const [time] = timePart.split("."); // Remove milliseconds
+      const isoString = `${year}-${month}-${day}T${time}Z`;
+      date = new Date(isoString);
+    } else {
+      date = new Date(dateString);
+    }
+
+    if (isNaN(date.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)